### ISSUE TITLE
Terminal tab actions should be displayed on focus, in addition to hover

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#122014]
Repro:

1. Open a terminal
2. Run Focus Terminal Tabs View command
3. The actions should show up

<img width="276" alt="Screen Shot 2021-04-23 at 5 17 53 AM" src="https://user-images.githubusercontent.com/2193314/115869900-644bc600-a3f3-11eb-91d2-771ef9352758.png">

SCM behaves as expected:

<img width="362" alt="Screen Shot 2021-04-23 at 5 17 58 AM" src="https://user-images.githubusercontent.com/2193314/115869906-66158980-a3f3-11eb-90f5-9ffa4659a36b.png">
